review.py

```diff
@@ -0,0 +1,164 @@
+        self.defense = defense
+        self.inventory = {}
+        self.equipped = {"head": None, "neck": None, "chest": None,
+                         "arm_r": None, "arm_l": None, "hands": None,
+                         "hand_r": None, "hand_l": None, "waist": None,
+                         "legs": None, "feet": None}
+
+    def adjust_stats(self, item, remove=False):
+        is_weapon = isinstance(item, Weapon)
+        if remove is True:
+            if is_weapon:
+                self.phys_attack -= item.attack_boost
+                self.accuracy -= item.accuracy_boost
+            else:
+                self.hp -= item.hp_boost
+                self.mp -= item.mp_boost
+                self.defense -= item.defense_boost
+        else:
+            if is_weapon:
+                self.phys_attack += item.attack_boost
+                self.accuracy += item.accuracy_boost
+            else:
+                self.hp += item.hp_boost
+                self.mp += item.mp_boost
+                self.defense += item.defense_boost
+
+    def equip_item(self, item):
+        equippable = self.can_equip(item)
+        if item not in self.inventory.keys():
+            print("You don't have that item.")
+        if equippable is True:
+            slot = item.slot
+            equipment = self.equipped[slot]
+            if equipment is not None:
+                while True:
+                    inp = input(
+                        f"Would you like to replace {equipment} with {item}? y/n:\n")
+                    if inp == "y":
+                        self.adjust_stats(equipment, True)
+                    elif inp == "n":
+                        print("The item stays in your inventory.")
+                        break
+                    else:
+                        print("Please answer with \"y\" or \"n\".")
+                        continue
+            else:
+                self.equipped[slot] = item
+                del self.inventory[item]
+                self.adjust_stats(item)
+
+        else:
+            print(equippable)
+
+    def can_equip(self, item):
+        equippable = None
+        if item.function == "equip":
+            equippable = True
+        else:
+            equippable = "That item cannot be equipped."
+        if equippable is True and self.level >= item.level:
+            equippable = True
+        elif equippable is True:
+            equippable = "You are not a high enough level to equip that item."
+        return equippable
+
+    def display_stats(self):
+        print(f"{self.name}'s stats:")
+        print(f"Level: {self.level}")
+        print(f"Dream Level: {self.dream_level}")
+        print(f"Hit Points: {self.hp}")
+        print(f"Magic Points: {self.mp}")
+        print(f"Physical Attack: {self.phys_attack}")
+        print(f"Accuracy: {self.accuracy}")
+        print(f"Defense: {self.defense}")
+        print(f"Inventory:", end=" ")
+
+    def display_inventory(self):
```

Comment:
The name of this method alone implies that `Inventory` kind of wants to be a class